docs/events.md

```diff
@@ -407,7 +407,7 @@ class ContentCategoriesAccepted extends ClientEvent {
 
 ### ClientEventSucceeded
 
-Event created to inform the client that a particular "fire and forget" event, like ie. `UserReactionChanged`, was successfuly processed by the engine.
+Event created to inform the client that a particular "fire and forget" event, like ie. `UserReactionChanged`, was successfully processed by the engine.
```

Comment:
I didn't notice this line before, but this kind of events are arguably "fire and forget". If `UserReactionChanged` fails we should not show it as green. Similarly for other events.

Reply:
I've updated docs a bit and removed the "fire and forget" bit.